jenkins-s2i/plugins.txt

```diff
@@ -1,168 +1,169 @@
-maven-plugin:3.0
-mercurial:2.2
-metrics:3.0.0
+matrix-auth:2.4.2
+matrix-project:1.14
+maven-plugin:3.4
+mercurial:2.8
+metrics:4.0.2.5
 momentjs:1.1.1
-mq-notifier:1.2.1
+mq-notifier:1.2.9
 multibranch-scan-webhook-trigger:1.0.5
 multiple-scms:0.6
 nexus-artifact-uploader:2.10
-nexus-jenkins-plugin:3.0.20180214-134325.e135900
-openshift-client:1.0.9
-openshift-login:1.0.8
-openshift-pipeline:1.0.54
-openshift-sync:1.0.10
-p4:1.3.8
-pam-auth:1.3
+nexus-jenkins-plugin:3.6.20190722-122200.83d1447
+openshift-client:1.0.32
+openshift-login:1.0.23
+openshift-pipeline:1.0.56
+openshift-sync:1.0.44
+p4:1.10.3
+pam-auth:1.5.1
 parameterized-trigger:2.35.2
 perforce:1.3.9
 pipeline-aggregator-view:1.8
-pipeline-build-step:2.7
+pipeline-build-step:2.9
 pipeline-github-lib:1.0
-pipeline-graph-analysis:1.6
+pipeline-graph-analysis:1.10
 pipeline-input-step:2.8
 pipeline-milestone-step:1.3.1
-pipeline-model-api:1.2.8
+pipeline-model-api:1.3.9
 pipeline-model-declarative-agent:1.1.1
-pipeline-model-definition:1.2.8
-pipeline-model-extensions:1.2.8
-pipeline-rest-api:2.10
+pipeline-model-definition:1.3.8
+pipeline-model-extensions:1.3.8
+pipeline-rest-api:2.12
 pipeline-stage-step:2.3
-pipeline-stage-tags-metadata:1.2.8
-pipeline-stage-view:2.10
-pipeline-utility-steps:2.0.2
-plain-credentials:1.4
-plot:2.0.3
-postbuildscript:2.6.0
+pipeline-stage-tags-metadata:1.3.8
+pipeline-stage-view:2.12
+pipeline-utility-steps:2.3.0
+plain-credentials:1.5
+plot:2.1.1
+postbuildscript:2.8.1
 preSCMbuildstep:0.3
-project-inheritance:1.5.3
-promoted-builds:2.27
+project-inheritance:2.0.0
+promoted-builds:3.3
 pubsub-light:1.12
-rabbitmq-consumer:2.2
+rabbitmq-consumer:2.8
 radiatorviewplugin:1.29
-rebuild:1.29
-resource-disposer:0.8
-ruby-runtime:0.12
-run-condition:1.0
-scm-api:2.2.6
-script-security:1.43
-slack:2.3
-sse-gateway:1.15
-ssh-credentials:1.13
-structs:1.14
-subversion:2.10.1
+rebuild:1.31
+resource-disposer:0.13
+run-condition:1.2
+scm-api:2.6.3
+script-security:1.64
+slack:2.37
+sse-gateway:1.19
+ssh-credentials:1.17.1
+structs:1.20
+subversion:2.12.2
 summary_report:1.15
-support-core:2.32
-timestamper:1.8.9
-token-macro:2.3
-variant:1.1
-view-job-filters:1.27
-vsphere-cloud:1.1.11
-workflow-aggregator:2.5
-workflow-api:2.26
-workflow-basic-steps:2.6
-workflow-cps-global-lib:2.9
-workflow-cps:2.46
-workflow-durable-task-step:2.16
-workflow-job:2.17
-workflow-multibranch:2.17
-workflow-remote-loader:1.4
-workflow-scm-step:2.6
-workflow-step-api:2.14
-workflow-support:2.18
-ws-cleanup:0.34
-xunit:1.102
+support-core:2.67
+timestamper:1.10
+token-macro:2.8
+variant:1.3
+view-job-filters:2.1.1
+vsphere-cloud:2.20
+workflow-aggregator:2.6
+workflow-api:2.35
+workflow-basic-steps:2.16
+workflow-cps-global-lib:2.14
+workflow-cps:2.72
+workflow-durable-task-step:2.30
+workflow-job:2.32
+workflow-multibranch:2.20
+workflow-remote-loader:1.5
+workflow-scm-step:2.9
+workflow-step-api:2.20
+workflow-support:3.3
+ws-cleanup:0.37
+xunit:2.3.5
+greenballs:1.15
```

Comment:
@ckavili Dont' we have the users add `greenballs:1.15` in as part of the exercise? So we may not want to include this here (as otherwise that part of the exercise isn't useful anymore)

Reply:
ah you are right!! I was going to through the exercises based on the docs to check if everything is fine, then I sent the PR and totally missed it. Just corrected, thanks! 👍